README.md

```diff
@@ -4,21 +4,105 @@
 ## What is ocdev?
 OpenShift Command line for Developers
 
+## Pre-requisites
+To use ocdev you need access to OpenShift instance and OpenShift CLI installed on your local machine (`oc` command in your $PATH).
+
+### OpenShift instance
+The easiest way to get started with OpenShift is [Minishift](https://docs.openshift.org/latest/minishift/index.html).
+
```

Comment:
Should we also point that ocdev will work with other OS instances?
Have we tried it on OSO?

Reply:
Yes, I've been testing it with OpenShift Online and it works without any problems.